cranelift/filetests/src/test_run.rs

```diff
@@ -85,8 +61,7 @@ fn is_isa_compatible(
 
     match (host_arch, requested_arch) {
         (host, requested) if host == requested => {}
-        (Architecture::Riscv64(host), Architecture::Riscv64(req))
-            if is_riscv64_compatible(host, req) => {}
+        (Architecture::Riscv64(_), Architecture::Riscv64(_)) => {}
```

Comment:
Should we still be doing a "subset of features required are present" test here?

Reply:
We can, but it seems somewhat redundant since we also do it down below.

And in practice we pretty much always use `riscv64` which (I think) gets parsed to the most generic form of `Riscv64Architecture::Riscv64` instead of the more specific `Riscv64gc` or `Riscv64imac`. Using the more generic form forces us to check features individually anyway.

Reply:
Ah, right, I missed the code below here that actually validates needed flags. All good then.